distributed/cli/dask_spec.py

```diff
@@ -10,7 +10,7 @@
 from distributed.deploy.spec import run_spec
 
 
-@click.command(context_settings=dict(ignore_unknown_options=True))
+@click.command(name="spec", context_settings=dict(ignore_unknown_options=True))
```

Comment:
Should we add help texts to the command and options given that we now expose `dask spec`?

Reply:
We could use `dask spec` in `ssh`, but I'm inclined to leave things as they are.  `python -m` is more reliable / fool-proof.  We could use `remote_python.rsplit("/", 1)[0] + "/dask spec"` but this could fail if, for example, there was a `python2` and `python3` binary in the same location.  I'm inclined to not get fancy here.